src/HH4b/processors/bbbbSkimmer.py

```diff
@@ -622,35 +651,29 @@ def process(self, events: ak.Array):
             add_selection("ak4_jetveto", cut_jetveto, *selection_args)
 
         if self._region == "signal":
-            # >=2 AK8 jets
-            add_selection("ak8_numjets", (ak.num(fatjets) >= 2), *selection_args)
+            legacy = "particleNetLegacy_mass" in fatjets.fields
 
-            # >=2 AK8 jets with pT>250
-            # TODO: check if fatjet passes pt cut in any of the JEC variations
-            cut_pt = (
-                np.sum(ak8FatJetVars["ak8FatJetPt"] >= self.preselection["fatjet_pt"], axis=1) >= 2
-            )
-            add_selection("ak8_pt", cut_pt, *selection_args)
+            # >=2 AK8 jets passing selections
+            add_selection("ak8_numjets", (ak.num(fatjets) >= 2), *selection_args)
 
-            # >=2 AK8 jets with mSD>60 or mReg>60
-            # TODO: check if fatjet passes mass cut in any of the JMS/R variations
-            cut_mass = (
-                np.sum(
-                    (ak8FatJetVars["ak8FatJetMsd"] >= self.preselection["fatjet_msd"])
-                    | (ak8FatJetVars["ak8FatJetPNetMass"] >= self.preselection["fatjet_mreg"])
-                    | (
-                        ak8FatJetVars["ak8FatJetPNetMassLegacy"] >= self.preselection["fatjet_mreg"]
-                    ),
-                    axis=1,
+            # >=1 bb AK8 jets (ordered by TXbb) with TXbb > 0.8
+            if not legacy:
+                cut_txbb = (
+                    np.sum(bbFatJetVars["bbFatJetPNetXbb"] >= self.preselection["Txbb0"], axis=1)
```

Comment:
@rkansal47 I think this is a bug: `bbFatJetPNetXbb` -> `bbFatJetPNetTXbb`?

Reply:
yup, fixed. luckily shouldn't affect the new jobs